Sources/Error Handling/BackendError.swift

```diff
@@ -49,9 +49,10 @@ extension BackendError {
     }
 
     static func missingReceiptFile(
+        _ receiptURL: URL?,
         file: String = #fileID, function: String = #function, line: UInt = #line
     ) -> Self {
-        return .missingReceiptFile(.init(file: file, function: function, line: line))
+        return .missingReceiptFile(receiptURL, .init(file: file, function: function, line: line))
```

Comment:
yesss! this seems so obvious in retrospective but I never thought about doing it